cmd/agent/main.go

```diff
@@ -275,11 +275,16 @@ func buildProbe(logger *zap.SugaredLogger, probeJSON string) *readiness.Probe {
 		logger.Fatalw("Agent failed to parse readiness probe", zap.Error(err))
 		panic("Agent failed to parse readiness probe")
 	}
-	return readiness.NewProbe(coreProbe)
+	newProbe := readiness.NewProbe(coreProbe)
+	newProbe.InitialDelaySeconds = 10
+	return newProbe
 }
 
 func buildServer(ctx context.Context, port string, userPort string, loggerArgs *loggerArgs, batcherArgs *batcherArgs,
 	probeContainer func() bool, logging *zap.SugaredLogger) (server *http.Server, drain func()) {
+
+	logging.Info("Building server ", userPort)
+	logging.Info("Port ", port)
```

Comment:
I think you need to use` logging.Infof` to format the log here, the two lines can also be combined into one.

Reply:
Sure will do.